src/styles/main.css

```diff
@@ -0,0 +1,4 @@
+@import "abstracts/variables.css";
+
+@import "base/reset.css";
+@import "base/typography.css";
```

Comment:
@HoangLiarPhuongLe
I think you should have an index file in each style folder (abstracts, base,...) to import the files there, and here you only need to import that file to keep it organized.